go/billing/models.py

```diff
@@ -50,6 +50,11 @@ class Account(models.Model):
                     "in the calculation to determine when low-credits "
                     "notifications should be sent."))
 
+    is_developer = models.BooleanField(
+        default=False,
+        help_text=_("Determines whether or not the billing account is a "
+                    "developer account."))
```

Comment:
I thought that perhaps we would do this as a feature flag on our ``go.vumitools.account`` objects?